### PR TITLE
Clair chart changes for security integration

### PR DESCRIPTION
**Pull request by dheeth**
- Default values as well as config changes

### CODE REVIEW

File: charts/clair/templates/deployment.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ spec:
       - name: pg-ready-wait
         image: postgres:11.3
         command: [ "sh", "-c",
-          "until pg_isready -h {{ .Values.config.postgresServiceName }} -p 5432;
+          "until pg_isready -h {{ .Values.config.postgresHost }} -p 5432;
```
Comment (nishant-d) [inline charts/clair/templates/deployment.yaml:38]: user `Values.config.postgresPort`

File: charts/clair/values.yaml [hunk 2]
```diff
@@ -46,11 +46,8 @@ podLabels:
   component: clair
 resources:
   limits:
-    cpu: 200m
-    memory: 1500Mi
-  requests:
-    cpu: 100m
-    memory: 500Mi
+    cpu: 500m
+    memory: 3Gi
```
Comment (nishant-d) [inline charts/clair/values.yaml:50]: no request?

**Review by nishant-d — CHANGES_REQUESTED** [review]

File: charts/clair/templates/deployment.yaml (hunk 1 above)
Comment (dheeth) [reply charts/clair/templates/deployment.yaml]: Done

File: charts/clair/values.yaml (hunk 2 above)
Comment (dheeth) [reply charts/clair/values.yaml:50]: Added requests

**Review by prakarsh-dt — APPROVED** [review]

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**